### PR TITLE
fix: Copy button doesn't work in containers section

### LINKED ISSUE #1736: Copy button doesn't work in containers section on Windows
### Bug description

When there are no containers, I am presented with the following screen -
image

When I click the copy button to copy the command `podman run quay.io/podman/hello`, it doesn't work. After pasting it, the clipboard doesn't contain anything.

### Operating system

Microsoft Windows 10 Pro 10.0.19045 N/A Build 19045

### Version

0.12.0

### Steps to reproduce

1. Install Podman desktop on Windows
2. Open Podman Desktop
3. Go to Containers.
4. Click on Copy output

### Relevant log output

```shell
N/A
```

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by jeffmaury**
Fixes #1736

### What does this PR do?

Fix the copy to clipboard feature for production bits

### What issues does this PR fix or reference?

Fixes #1736 
### How to test this PR?

- Generate the production bits
- Launch
- Go the empty containers tab and copy the instruction to clipboard
- Ensure data is in clipboard

### CODE REVIEW

File: packages/renderer/src/lib/ui/EmptyScreen.spec.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+import EmptyScreen from './EmptyScreen.svelte';
+
+test('Expect copy in clipboard', async () => {
+  render(EmptyScreen, {commandline: 'podman run hello:world'});
+  const button = screen.getByRole('button');
+  expect(button).toBeInTheDocument();
+  expect(button).toBeEnabled();
+  button.click();
+  const div = screen.getByTestId('copyTextDivElement');
+  expect(div).toBeInTheDocument();
```
Comment (benoitf) [inline packages/renderer/src/lib/ui/EmptyScreen.spec.ts:32]: Shouldn't the test expect that the value is in the clipboard ?

File: packages/renderer/src/lib/ui/EmptyScreen.spec.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+import { test, expect } from 'vitest';
+import { render, screen } from '@testing-library/svelte';
+import EmptyScreen from './EmptyScreen.svelte';
+
+test('Expect copy in clipboard', async () => {
+  render(EmptyScreen, {commandline: 'podman run hello:world'});
+  const button = screen.getByRole('button');
+  expect(button).toBeInTheDocument();
+  expect(button).toBeEnabled();
+  button.click();
```
Comment (benoitf) [inline packages/renderer/src/lib/ui/EmptyScreen.spec.ts]: we need to use fireEvent for svelte

File: packages/renderer/src/lib/ui/EmptyScreen.spec.ts (hunk 1 above)
Comment (benoitf) [inline packages/renderer/src/lib/ui/EmptyScreen.spec.ts]: Test should expect that the clipboard method is called

File: packages/renderer/src/lib/ui/EmptyScreen.spec.ts (hunk 2 above)
Comment (benoitf) [reply packages/renderer/src/lib/ui/EmptyScreen.spec.ts]: await fireEvent.click(button);

**Review by benoitf — COMMENTED** [review]
It looks odd that you don't need to mock the window copy clipboard method. Maybe it's because the click method is not performing the click so we're not trying to call window method

File: packages/renderer/src/lib/ui/EmptyScreen.spec.ts [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+import '@testing-library/jest-dom';
+import { test, expect, vi } from 'vitest';
+import { fireEvent, render, screen } from '@testing-library/svelte';
+import EmptyScreen from './EmptyScreen.svelte';
+
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/* eslint-disable @typescript-eslint/no-empty-function */
+
+test('Expect copy in clipboard', async () => {
+  console.log('start test');
```
Comment (benoitf) [inline packages/renderer/src/lib/ui/EmptyScreen.spec.ts]:
```suggestion

```

File: packages/renderer/src/lib/ui/EmptyScreen.spec.ts [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+  const mock = vi.fn().mockImplementation(() => {});
+  (window as any).clipboardWriteText = mock;
+  render(EmptyScreen, { commandline: 'podman run hello:world' });
+  const button = screen.getByRole('button');
+  expect(button).toBeInTheDocument();
+  expect(button).toBeEnabled();
+  await fireEvent.click(button);
+  expect(mock).toBeCalledTimes(1);
```
Comment (benoitf) [inline packages/renderer/src/lib/ui/EmptyScreen.spec.ts]: toBeCalledWith('podman run hello:world')
so we're sure that we want to copy the string that we're giving

Comment (benoitf) [reply packages/renderer/src/lib/ui/EmptyScreen.spec.ts]: Also toBeCalledTimes(1) or toBeCalled() is the same

**Review by benoitf — COMMENTED** [review]
Code LGTM just small nits to fix 👍
I'll check on macOS tomorrow but should be working as well ( or if someone else check on macOS before)

**Review by benoitf — APPROVED** [review]
Works fine for me on macOS 👍 
please remove the `console.log('start test');` in the test before merging

https://user-images.githubusercontent.com/436777/227456286-32481cea-c2e3-40e0-8dbc-2eb68871fb45.mp4

File: packages/renderer/src/lib/ui/EmptyScreen.spec.ts [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+test('Expect copy in clipboard', async () => {
+  console.log('start test');
+  const mock = vi.fn().mockImplementation(() => {});
+  (window as any).clipboardWriteText = mock;
+  render(EmptyScreen, { commandline: 'podman run hello:world' });
+  const button = screen.getByRole('button');
+  expect(button).toBeInTheDocument();
+  expect(button).toBeEnabled();
+  await fireEvent.click(button);
+  expect(mock).toBeCalledWith('podman run hello:world');
```
Comment (benoitf) [inline packages/renderer/src/lib/ui/EmptyScreen.spec.ts:35]: so it's strange it seems it doesn't give this string in the test

**Review by lstocchi — APPROVED** [review]
Works fine on WIndows 🚀

**Pull request merged**